build.cake

```diff
@@ -0,0 +1,106 @@
+#addin "Cake.FileHelpers"
+
+var target = Argument("target", "Default");
+
+var buildDir = "./build/";
```

Comment:
Perhaps we could try using the [`DirectoryPath`](http://cakebuild.net/api/Cake.Core.IO/DirectoryPath/2EE6D32A) class instead of plain strings. This will allow us to convert this path to a full, absolute path using [`MakeAbsolute`](http://cakebuild.net/api/Cake.Core.IO/DirectoryPath/2FAC60BD).